engines/scumm/scumm.cpp

```diff
@@ -1420,6 +1421,10 @@ void ScummEngine::setupScumm(const Common::String &macResourceFile) {
 	}
 
 	_res->setHeapThreshold(400000, maxHeapThreshold);
+#else
+	// RAM is cheap, disk I/O isn't... helps with retaining the resources in COMI and similar
```

Comment:
Average Atari system is still out of reach for my work, unfortunately. The absolute minimum is set to a 32 MHz CPU (ideally 50 MHz) and 64 MB of so called Fast/TT RAM (this is easily available via addon cards).

However I have seen reports that _some_ games are playable (with native MIDI instead of emulation) even on basic configs like 16 MHz without too much RAM (that was before this commit) so I might reconsider decreasing this huge cache; see my TODO: https://github.com/scummvm/scummvm/pull/4687/files#diff-7ce93d26e2f2ac33135484bf47a2063bf4bb1b3732195c40797f29098d13611fR398-R399

Reply:
Maybe in a separate PR we could add a new function to OSystem that wraps around `SDL_GetSystemRAM` and use that to decide how much memory should be allocated for the cache?

Reply:
Sounds good. I wonder how many engines have hard limits like those in scumm set. But maybe other engines don't have such a concept of loading/releasing resources, don't know...